chromatic/rainbows/readers/x1dints.py

```diff
@@ -231,11 +255,11 @@ def from_x1dints(rainbow, filepath, order=None, **kw):
                     """
                     )
 
-                # define a complete list of
-                non_spectrum_extensions = [
-                    x for x in ["PRIMARY", "SCI", "INT_TIMES", "ASDF"] if x in hdu
-                ]
-                non_asdf_non_spectrum_extensions = non_spectrum_extensions[:-1]
+            # define a complete list of
```

Comment:
It's weird that the extensions would change between different segments of the same observation, but if this fixes the problem, let's go with it!

Reply:
This is a version issue, one of the files hasn't been updated to 1.7.2 for some reason